emoji-ui/src/components/emojiItem.css

```diff
@@ -0,0 +1,18 @@
+.Emoji-container {
```

Comment:
@lisatsewang Have you looked into compiling these from [CSSQL](https://github.com/mcnuttandrew/cssql)?

Reply:
i have not. i will read up on `CSSQL`.

Reply:
lol! maybe not 🤣